### PR TITLE
Include RunTests.exe and xunit.console.x86.exe in .NET event collection

### PR DESCRIPTION
**Pull request by sharwell**
Captures logs necessary to diagnose crashes during test runs.

### CODE REVIEW

**Comment by sharwell** [comment]
📝 The build failure was caused by #33516

File: src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs [hunk 1]
```diff
@@ -67,14 +67,14 @@ private static void FirstChanceExceptionHandler(object sender, FirstChanceExcept
                 var assemblyDirectory = GetAssemblyDirectory();
                 var testName = CaptureTestNameAttribute.CurrentName ?? "Unknown";
                 var logDir = Path.Combine(assemblyDirectory, "xUnitResults", "Screenshots");
-                var baseFileName = $"{DateTime.Now:HH.mm.ss}-{testName}-{eventArgs.Exception.GetType().Name}";
+                var baseFileName = $"{DateTime.UtcNow:HH.mm.ss}-{testName}-{eventArgs.Exception.GetType().Name}";
```
Comment (RikkiGibson) [inline src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs]: What's the impact of this? Just improves the experience of looking for a log for a particular integration test failure?

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs]: We capture .NET and Watson event logs, and the timestamps in those files are UTC. Sometimes the events occurred during previous runs, so having the same dates for screenshots makes it clear if it's part of this run or a previous run.

File: src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs [hunk 2]
```diff
@@ -107,6 +108,7 @@ internal static class EventLogCollector
                 "vstest.executionengine.clr20.exe",
                 "VSTest.executionEngine.exe",
                 "VSTest.executionEngine.x86.exe",
```
Comment (jaredpar) [inline src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs]: While you're here can you delete vbc2.exe. The csc2.exe has already been deleted.

File: src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs [hunk 3]
```diff
@@ -107,6 +108,7 @@ internal static class EventLogCollector
...
+                "xunit.console.x86.exe",
```
Comment (jaredpar) [inline src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs]: The 64 bit version of other testing hosts were included here. Consider adding the 64 bit version of xunit. Don't feel strongly either way here.

File: src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs (hunk 2 above)
Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs]: ➡️ Submitted in #33520

File: src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs (hunk 3 above)
Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/EventLogCollector.cs]: ➡️ Submitted in #33520

**Pull request merged**